src/ChainSafe.Gaming.SygmaClient/Types/ConfigUrl.cs

```diff
@@ -0,0 +1,21 @@
+namespace ChainSafe.Gaming.SygmaClient.Types
+{
+    public static class ConfigUrl
+    {
+        public static readonly string Devnet = "https://chainbridge-assets-stage.s3.us-east-2.amazonaws.com/shared-config-dev.json";
```

Comment:
You can use `const` instead of `static readonly` for strings.

Reply:
Will do, thank you!